spec/models/team_spec.rb

```diff
@@ -147,6 +147,80 @@
     end
   end
 
+  describe 'limit of coaches' do
+    let(:team) { create :team }
+    let(:new_user) { create :user }
+    let(:coach_attributes) { [{ name: 'coach', team_id: team.id, user_id: new_user.id }] + 3.times.map { { name: 'coach', team_id: team.id, user_id: create(:user).id } } }
+    let(:new_coach_as_student) { { name: 'student', team_id: team.id, user_id: new_user.id } }
+    let(:fifth_new_coach) { { name: 'coach', team_id: team.id, user_id: create(:user).id } }
+    let(:remove_coach) { { name: 'coach', team_id: team.id, user_id: create(:user).id, _destroy: true } }
+
+    context 'when team has no coaches yet' do
+      it 'allows to add 2 new coaches' do
```

Comment:
the description says 2, but it's about 4 in the example :)

Reply:
Done!